src/components/header/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import './header.css';
+
+function Header() {
+
+    const [userName, setUserName] = useState('');
+
+    return (
+        <section id="header">
+            <nav>
+                <div>
+                    <NavLink to="/">Home</NavLink> {' '}
+                    <NavLink to="/income">Income</NavLink> {' '}
+                    <NavLink to="/expenses">Expenses</NavLink> {' '}
+                    <NavLink to="/chart">Chart</NavLink> {' '}
+                    <NavLink to="/login">
+                        <button className="btn btn-primary">log-in</button>
```

Comment:
please stop adding these classes to the buttons, they are not needed

Reply:
Use your reusable component.